Contributing.md

```diff
@@ -0,0 +1,36 @@
+# Contributing to aqa-tests
+
+Thanks for your interest in this project.
+
+## Project description
+
+aqa-tests is the central project for AQAvit and contains the definition files for the AQA test suites
```

Comment:
Not sure if  capitalize `aqa-tests` would be better.

Reply:
I am matching it to the github repo name which is all lowercase (and how we did the aqa-test-tools Contributing.md file also)